src/components/alt-button/FavoriteButton.tsx

```diff
@@ -1,3 +1,5 @@
+// TODO(nkang) - turn into button component like RepostButton, add accessibility
```

Comment:
TODOs tend to get lost. Should we make a linear issue?

Reply:
Yeah I'll do that!

Reply:
oh amazing, you're tracking this! ill remove my previous comment

Reply:
yess sounds good. definitely want to make sure everything that goes into stems is properly accessible

Reply:
https://linear.app/audius/issue/AUD-1432/turn-favoritebutton-into-actual-button-element-like-repostbutton-add